molgenis_documentation/user_documentation.md

```diff
@@ -161,25 +139,26 @@ The fastest and easiest way to get MOLGENIS running on a machine, is using our c
 <!--
 TODO: Get this project to the molgenis repository, so I do not have to link to the Github of fleur
 -->
+
 [Download](https://github.com/fdlk/molgenis-cargo) the entire project from GitHub.
 
 **Setting your molgenis-server.properties**   
 MOLGENIS will try to find its property file at *<user_home>/.molgenis/omx/*. Create this folder, and create the molgenis-server.properties file. Open the file and write the following lines:
 
-> db_user=molgenis  
-> db_password=molgenis  
-> db_uri=jdbc:mysql://localhost/omx  
-> admin.password=admin  
-> user.password=admin  
+	db_user=molgenis	
```

Comment:
For cargo, you don't need to do this. The file is included in the project and copied when you run the clean target.